src/Parser/Priest/Discipline/Modules/Abilities.js

```diff
@@ -40,51 +45,15 @@ class Abilities extends CoreAbilities {
       },
       {
         spell: SPELLS.POWER_WORD_SHIELD,
-        name: `${SPELLS.POWER_WORD_SHIELD.name} outside Rapture`,
-        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: haste => 9 / (1 + haste),
-        castEfficiency: {
-          suggestion: true,
-          casts: castCount => castCount.casts - (castCount.raptureCasts || 0),
-          maxCasts: (cooldown, fightDuration, getAbility, parser) => {
-            const timeSpentInRapture = parser.modules.combatants.selected.getBuffUptime(SPELLS.RAPTURE.id);
-            const maxRegularCasts = calculateMaxCasts(cooldown, fightDuration - timeSpentInRapture);
-
-            return maxRegularCasts;
-          },
-        },
-      },
-      {
-        spell: SPELLS.POWER_WORD_SHIELD,
-        name: `${SPELLS.POWER_WORD_SHIELD.name} during Rapture`,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: haste => 9 / (1 + haste),
-        castEfficiency: {
-          suggestion: true,
-          extraSuggestion: `${SPELLS.POWER_WORD_SHIELD.name} may be cast without cooldown during Rapture.`,
-          casts: castCount => castCount.raptureCasts || 0,
-          maxCasts: (cooldown, fightDuration, getAbility, parser) => {
-            const gcd = 1.5 / (1 + parser.modules.haste.current);
-            const timeSpentInRapture = parser.modules.combatants.selected.getBuffUptime(SPELLS.RAPTURE.id);
-
-            const maxRaptureCasts = calculateMaxCasts(gcd, timeSpentInRapture);
-
-            return maxRaptureCasts;
-          },
+        cooldown: (haste, selectedCombatant) => {
+          if(selectedCombatant.hasBuff(SPELLS.RAPTURE.id)) {
+            return 1.5 / (1 + haste);
```

Comment:
Cast efficiency wont work if the cooldown is set to 0. Could make a cast efficiency specifically to track this but it would be a lot of duplicate code otherwise.